sqlx-postgres/src/type_info.rs

```diff
@@ -1152,7 +1152,10 @@ impl PartialEq<PgType> for PgType {
             true
         } else {
             // Otherwise, perform a match on the name
-            self.name().eq_ignore_ascii_case(other.name())
+            // If either name has no schema qualification, match only on the name part.
```

Comment:
I guess it is not possible with the current info (due to `search_path`) to create a fully correct eq method using pg_ type names. How did you decide on this matching?